std/conv.d

```diff
@@ -81,8 +81,16 @@ private auto convError(S, T)(S source, string fn = __FILE__, size_t ln = __LINE_
     if (source.empty)
         msg = "Unexpected end of input when converting from type " ~ S.stringof ~ " to type " ~ T.stringof;
     else
-        msg =  text("Unexpected '", source.front,
+    {
+        import std.range : ElementType;
+        ElementType!S el = source.front;
+
+        if (el == '\n')
+            msg = text("Unexpected '\\n' when converting from type " ~ S.stringof ~ " to type " ~ T.stringof);
+        else
+            msg =  text("Unexpected '", el,
```

Comment:
actually, there isn't. I searched through format and the only function which seems to do the job is formatChar [1] which is private and still calls formattedWrite.

[1] https://github.com/dlang/phobos/blob/master/std/format.d#L3027